NEWS.md

```diff
@@ -1,6 +1,11 @@
 # covidregionaldata 0.9.1
 
 In development release.
+
+## New features
+
```

Comment:
increment the dev version in the DESCRIPTION file

Reply:
changed to `Version: 0.9.1.9000` following [this guide](https://r-pkgs.org/release.html), is this correct? sorry still new to versioning